api/Onboard.go

```diff
@@ -0,0 +1,46 @@
+package api
+
+import (
+	"fmt"
+	"github.com/operate-first/opfcli/models"
+	"strings"
+)
+
+func (api *API) Onboard(path string, projectDisplayName string, disableLimitrange bool) error {
+	onboardRequest, err := models.OnboardRequestFromYAMLPath(path)
+
+	if err := api.CreateGroup(
+		onboardRequest.Teamname,
+		true,
+	); err != nil {
+		return err
+	}
+
+	if err := api.CreateRoleBinding(onboardRequest.Teamname, "admin"); err != nil {
+		return err
+	}
+
+	var tempQuota string
+	for i := 0; i < len(onboardRequest.Namespaces); i++ {
+		if onboardRequest.Namespaces[i].Quota != "" && onboardRequest.Namespaces[i].Quota != "_No response_" {
+			tempQuota = strings.ToLower(onboardRequest.Namespaces[i].Quota)
+			if err := api.ValidateQuota(tempQuota); err != nil {
+				return err
+			}
+		}
+		api.CreateNamespace(
+			onboardRequest.Namespaces[i].Name,
+			onboardRequest.Teamname,
+			projectDisplayName,
+			tempQuota,
+			disableLimitrange,
+			true,
+		)
+	}
+
+	if err != nil {
```

Comment:
this shouldn't be needed, you should be checking `err` for `nil` as you go, by this point this condition should always be `false`